client/state/data-getters/index.js

```diff
@@ -61,17 +61,21 @@ export const requestActivityActionTypeCounts = (
 	);
 };
 
-export const requestActivityLogs = ( siteId, filter, { freshness = 5 * 60 * 1000 } = {} ) => {
+export const requestActivityLogsId = ( siteId, filter ) => {
```

Comment:
Why do we need to export this function? It seems like a 'private' helper function to create only the Id for the API.

Reply:
[You can see it in use here](https://github.com/Automattic/wp-calypso/pull/40494/files#diff-90651e10bbc95d5c8b38fe246249c55fR53). To get access to this data we need the id, [which indexes the response in httpData map.](https://github.com/Automattic/wp-calypso/blob/master/client/state/data-layer/http-data.ts)